### PR TITLE
[DOCS] Fix rollup groups doc issue.

### PR DESCRIPTION
**Pull request by howardhuanghua**
Fix doc issue.

### CODE REVIEW

File: docs/reference/rollup/understanding-groups.asciidoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ experimental[]
 
 To preserve flexibility, Rollup Jobs are defined based on how future queries may need to use the data.  Traditionally, systems force
 the admin to make decisions about what metrics to rollup and on what interval.  E.g. The average of `cpu_time` on an hourly basis.  This
-is limiting; if, at a future date, the admin wishes to see the average of `cpu_time` on an hourly basis _and partitioned by `host_name`_,
+is limiting, in the future if the admin wishes to see the average of `cpu_time` on an hourly basis and partitioned by `host_name`,
```
Comment (jrodewig) [inline docs/reference/rollup/understanding-groups.asciidoc:10]: The suggested change isn't grammatically correct. Can you explain the problem you were attempting to solve with this?

**Review by jrodewig — CHANGES_REQUESTED** [review]
Thanks for raising this @howardhuanghua. However, I'm not sure if these changes are needed. In any case, there are some grammatical issues with the proposed change.

I left a related question. Once I get some more clarity about the problem you're trying to solve, I can provide more help.

Thanks again!

File: docs/reference/rollup/understanding-groups.asciidoc (hunk 1 above)
Comment (howardhuanghua) [reply docs/reference/rollup/understanding-groups.asciidoc:10]: Hi @jrodewig this is the original doc:
https://www.elastic.co/guide/en/elasticsearch/reference/master/rollup-understanding-groups.html
I think:
1. There should no comma after "if".
2. "host_name" format is not correct.
image
image

Comment (jrodewig) [inline docs/reference/rollup/understanding-groups.asciidoc:10]: Thanks for the clarification. I'm going to incorporate a suggestion that fixes the formatting without removing the semicolon.

```suggestion
is limiting; if, in the future, the admin wishes to see the average of `cpu_time` on an hourly basis _and_ partitioned by `host_name`,
```

Comment (howardhuanghua) [reply docs/reference/rollup/understanding-groups.asciidoc]: Hi @jrodewig thanks for the suggestion. I have updated the change.

**Review by jrodewig — APPROVED** [review]
LGTM. Thanks @howardhuanghua!

**Comment by jrodewig** [comment]
@elasticmachine ok to test

**Pull request merged**